cloud/amazon/s3_bucket.py

```diff
@@ -290,6 +300,39 @@ def destroy_bucket(connection, module):
         
     module.exit_json(changed=changed)
 
+def _create_bucket_ceph(connection, module, location):
+
+    name = module.params.get("name")
+
+    changed = False
+
+    try:
+        bucket = connection.get_bucket(name)
+    except S3ResponseError, e:
```

Comment:
We're aiming for python3 syntax-compilation compatibility for 2.2 so we'll need to change the catching of exceptions.  It looks like this module requires boto which requires python >= 2.6 so you can change to syntax like this:
``` python
except S3ResponseError as e:
```